med_inventory/templates/inventory_check.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Inventory Check</title>
+    <style>
+        /* Button styling */
+        button {
+            padding: 10px 15px;
+            font-size: 16px;
+            background-color: #4CAF50;
+            color: white;
+            border: none;
+            cursor: pointer;
+            border-radius: 5px;
+            margin-top: 20px;
+        }
+
+        button:hover {
+            background-color: #45a049;
+        }
+
+        /* Page styling */
+        body {
+            font-family: Arial, sans-serif;
+            max-width: 600px;
+            margin: 0 auto;
+            padding: 20px;
+        }
+
+        h1 {
+            color: #333;
+        }
+
+        select, button {
+            margin-top: 10px;
+            display: block;
+        }
+    </style>
+</head>
+<body>
+    <h1>Check Drug Inventory</h1>
+    
+    <form method="GET" action="{% url 'inventory_check' %}">
+        <label for="drug">Select Drug:</label>
+        <select id="drug" name="drug">
+            <option value="">--Select Drug--</option>
+            {% for drug in drugs %}
+                <option value="{{ drug.id }}"{% if selected_drug and selected_drug.id == drug.id %} selected{% endif %}>{{ drug.drug_name }}</option>
```

Comment:
I am able to click the button even when "---Select Drug---" is my option which throws an error message

Reply:
Right. you have to select a drug for it to not throw the error. i can change that if thats better

Reply:
Fixed the error in the latest push.